Sources/streams/DuplexStream.swift

```diff
@@ -234,7 +234,7 @@ open class DuplexStream<ReadType, WriteType>
 
 private class _DuplexReadStream<TI, TO> : ReadableStream<TI> {
   
-  unowned let parent : DuplexStream<TI, TO>
+  private var parent : DuplexStream<TI, TO>?
```

Comment:
This should carry an explanation. I thought that `DuplexStream` is owning its read & write streams. `unowned` feels right. Why/when is this an issue? It almost sounds like the read stream leaks out of that abstraction when it shouldn't (the sub-streams of Duplex should be completely private).